parm/convection_permitting/namelist.init_atmosphere

```diff
@@ -11,20 +11,22 @@
     config_nfglevels = ${nfglevels}
     config_nfgsoillevels = ${nfgsoillevels}
     config_nsoilcat      = 16
+    config_nvegopt       = 2
 /
 &data_sources
-    config_geog_data_path = ''
+    config_geog_data_path = '/mnt/lfs5/BMC/wrfruc/HRRRv5/geog/'
```

Comment:
we don't need line 17 to be defined in the workflow. It is only needed in the generation of static of invariant fields.

Reply:
If we do not have this path set to a valid data location, the ic task crashed:

```
----------------------------------------------------------------------
Beginning MPAS-init_atmosphere Error Log File for task       0 of     800
    Opened at 2024/11/06 06:49:43
----------------------------------------------------------------------

ERROR: Error reading topography tile /topo_ugwp_30s/00001-08640.00001-04320
ERROR: Error reading topography tile /topo_ugwp_2.5m/00001-08640.00001-04320
```

Reply:
Thanks for the information. This is a new requirement from the GSL's MPAS model.
We need to make the workflow run on other HPCs as well.
So a better way is to stage topo_ugwp* under FIXrrfs.  I can help with this.

Reply:
@hu5970 I repeated this case but found that we can still generate `init.nc` even when we cannot read the topopgraphy file.  This triggered a question: do we really need those highres topo_ugwp files at runtime?